tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ng.html

```diff
@@ -108,4 +109,35 @@
       (onResize)="redraw()"
     ></tb-line-chart>
   </ng-template>
+
+  <ng-template #tooltip let-tooltipData="data">
+    <table class="tooltip">
+      <thead>
+        <tr>
+          <th class="circle-header"></th>
+          <th>Run</th>
+          <th *ngIf="smoothingEnabled">Smoothed</th>
+          <th>Value</th>
+          <th>Step</th>
+          <th>Time</th>
+        </tr>
+      </thead>
+      <tbody>
+        <ng-container
+          *ngFor="let datum of tooltipData; trackBy: trackByTooltipDatum"
+        >
+          <tr class="tooltip-row">
+            <td class="tooltip-row-circle">
+              <span [style.backgroundColor]="datum.metadata.color"></span>
+            </td>
+            <td class="name">{{ datum.metadata.displayName }}</td>
+            <td>{{ datum.point.y }}</td>
```

Comment:
They are not resulting in a bug and I thought it would be nicer to use `datum.point.y` but I found myself having to explain the rationale a bit longer than ideal so I moved it to the third column for consistency.

Tests proved that they are logically the same.